CONTRIBUTING.md

```diff
@@ -61,7 +62,7 @@ Semantic conventions have some autogenerated components and additionally can do
 automatic style/spell correction. You can run all of this via:
 
 ```bash
-make fix
+npm run fix
```

Comment:
Can we keep `make` as the entry point, and just have it call `npm` ?

Reply:
Certainly. Done, PTAL.